pkg/collector/pipeline_test.go

```diff
@@ -29,7 +29,23 @@ import (
 
 var validNelReportPath = filepath.Clean("../pipelinetest/testdata/reports/valid-nel-report.json")
 
-func TestIgnoreNonPOST(t *testing.T) {
+func TestRespondsToOptionsRequest(t *testing.T) {
+	pipeline := collector.NewPipeline(pipelinetest.NewSimulatedClock())
+	request := httptest.NewRequest("OPTIONS", "https://example.com/upload", bytes.NewReader([]byte("")))
+	response := httptest.NewRecorder()
+	pipeline.ServeHTTP(response, request)
+	if want, got := "POST", response.Header().Get("Access-Control-Allow-Methods"); got != want {
```

Comment:
Does it work to change this to:

``` go
if want := "POST"; got := /* etc */; got != want {
```

I'm half-convinced that that's more readable.  What do you think?

Reply:
I'm completely convinced; I had that originally and it didn't compile. :(